website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx

```diff
@@ -38,6 +38,8 @@ To delete a key, find it in the list of keys and click its "Delete" button. Befo
 
 ~> **Important:** If any workspaces are still using a key when you delete it, they will be unable to clone modules from private repos until you assign them a new key. This might cause Terraform runs to fail.
```

Comment:
```suggestion
~> **Important:** If any workspaces are still using a key when you delete it, they can no longer clone modules from private repos until you assign them a new key. This inability might cause Terraform runs to fail.
```

A small suggested style change to avoid future tense and to avoid having a "This" with no subject.